systemtest/src/main/java/io/strimzi/systemtest/utils/VerificationUtils.java

```diff
@@ -0,0 +1,463 @@
+     * @param namespaceName Namespace name where config maps are located
+     * @param clusterName Name of the cluster linked with configmaps - used for config prefix
+     * @param appName Name of the app linked with the config maps
+     * @param additionalClusterName Name of the second cluster - used mainly for source + target cluster verification
+     */
+    public static void verifyConfigMapsLabels(String namespaceName, String clusterName, String appName, String additionalClusterName) {
+        LOGGER.info("Verifying labels for Config maps");
+
+        kubeClient(namespaceName).listConfigMaps()
+            .forEach(cm -> {
+                LOGGER.info("Verifying labels for CM {}", cm.getMetadata().getName());
+                if (cm.getMetadata().getName().equals(clusterName.concat("-connect-config"))) {
+                    assertThat(cm.getMetadata().getLabels().get("app"), is(nullValue()));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_KIND_LABEL), is(KafkaConnect.RESOURCE_KIND));
+                } else if (cm.getMetadata().getName().contains("-mirror-maker-config")) {
+                    assertThat(cm.getMetadata().getLabels().get("app"), is(nullValue()));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_KIND_LABEL), is(KafkaMirrorMaker.RESOURCE_KIND));
+                } else if (cm.getMetadata().getName().contains("-mirrormaker2-config")) {
+                    assertThat(cm.getMetadata().getLabels().get("app"), is(nullValue()));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_KIND_LABEL), is(KafkaMirrorMaker2.RESOURCE_KIND));
+                } else if (cm.getMetadata().getName().equals(clusterName.concat("-kafka-config"))) {
+                    assertThat(cm.getMetadata().getLabels().get("app"), is(appName));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_KIND_LABEL), is(Kafka.RESOURCE_KIND));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_CLUSTER_LABEL), is(clusterName));
+                } else if (cm.getMetadata().getName().equals(additionalClusterName.concat("-kafka-config"))) {
+                    assertThat(cm.getMetadata().getLabels().get("app"), is(appName));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_KIND_LABEL), is(Kafka.RESOURCE_KIND));
+                    assertThat(cm.getMetadata().getLabels().get(Labels.STRIMZI_CLUSTER_LABEL), is(additionalClusterName));
+                } else {
+                    LOGGER.info("CM {} is not related to current test", cm.getMetadata().getName());
+                }
+            }
+        );
+    }
+
+    /**
+     * Verifies metadata labels for service
+     * @param namespaceName Namespace name where service is located
+     * @param clusterName Name of the cluster linked with service - used for prefix
+     * @param nameLabel "Name" label in the service metadata
```

Comment:
`nameLabel` is again suffix which determines the component name, right? What about passing the component name instead of name label?

Reply:
That is true, i have it inconsistent here. Changed